Improve bug report (#28362)
metabase/metabase: .github/ISSUE_TEMPLATE/bug_report.md

assignees: ''

---
**Read this before creating a bug report**

> - in case any of the sections is incomplete, we might close the bug report as we won't be able to reproduce it

Reviewer: ```suggestion
- if any of the sections are incomplete, we might close the bug report since we won't be able to reproduce it
```

> - have you checked if the bug has already been reported? as we close duplicate reports, please search on the issue tracker if the problem has been reported before

Reviewer: ```suggestion
- have you checked if the bug has already been reported? As we close duplicate reports, please search on the issue tracker if the problem has been reported before
```

Reviewer 2: Was this meant to sound like - search **because** we will close if it is a duplicate report?

`As we close duplicate reports` is confusing me a bit.

- use as many screenshots or videos as possible to describe what you're seeing, all visual representations speak for themselves

**Describe the bug**
A clear and concise description of what the bug is.

**Logs**

> Please include javascript console (open the browser dev tools and check the "console" tab) and server logs (Metabase logs that you can get from settings->admin->troubleshooting->logs) around the time this bug occurred. For information about how to get these, consult our [bug troubleshooting guide](https://metabase.com/docs/latest/troubleshooting-guide/bugs.html)

Reviewer 2: ```suggestion
Please include JavaScript console logs (open the browser dev tools and check the "console" tab) as well as the server logs (Metabase logs that you can get from Settings -> Admin -> Troubleshooting -> Logs) around the time this bug occurred. For information about how to get these, consult our [bug troubleshooting guide](https://metabase.com/docs/latest/troubleshooting-guide/bugs.html)
```

**To Reproduce**

> Steps to reproduce the behavior (if you can use the sample dataset to reproduce this bug, we will be able to find the issue faster):

Reviewer 2: ```suggestion
Steps to reproduce the behavior (if you can reproduce the bug using the Sample Database, we will find the issue faster):
```
